chroot-bin/runsommelier

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh -eu
```

Comment:
Is there any reason someone would want to run sommelier directly?  seems like we should be installing sommelier to /usr/bin and this script to /usr/local/bin as just "sommelier", and then putting the absolute path in the chained execution below

Reply:
We could follow the approach taken by the vm-tools in chromiumos/Debian and install the binary as `sommelier.real` and the script as `sommelier`. If the former is installed in `/usr/local/lib` then it would not interfere with command completion.

Anyway, I agree that calling the runnable script `sommelier` instead of `runsommelier` makes more sense.

Reply:
That sounds good to me.

Reply:
Resolved with two separate commits so as to allow alternate solutions with the same renaming of `runsommelier` to `sommelier`:

- commit 98d12b021f778896b8d198887833326fae2ba94e
- commit 7fb307182c49aea6a8adddb09e7106bb0411491e